package.json

```diff
@@ -46,9 +46,12 @@
 		"webpack-merge": "^4.1.1"
 	},
 	"engines": {
-		"node": ">=8"
+		"node": "9.3.0",
+		"npm": "5.5.1"
 	},
 	"scripts": {
+		"start": "node server.js",
```

Comment:
Seems to me that `yarn start` basically does the same thing.

Reply:
`yarn start` should be running this command, from my understanding its just shorthand for `yarn run start`